frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -137,33 +144,19 @@ export const dashboardLogic = kea<dashboardLogicType>({
                         return null
                     }
 
-                    const refreshStartTime = performance.now()
                     const dashboardQueryId = uuid()
+                    actions.setInitialLoadTimer(action, dashboardQueryId)
```

Comment:
I'm tempted to nit on the name :) I think this is `loadingDashboardItemsStarted` but not blocking

Reply:
Interestingly while convincing myself of the order of operations here I spotted that when loading a dashboard on the home page this is called twice but not when loading a dashboard on its own page

Reply:
Facinating. I'll take a look at home page separately!